subiquitycore/utils.py

```diff
@@ -176,3 +176,10 @@ def disable_subiquity():
                  "snap.subiquity.subiquity-service.service",
                  "serial-subiquity@*.service"])
     return
+
+
+def child_matches(items, **kw):
```

Comment:
Is the term "child" really relevant here? If I understand correctly, this function returns a subset of the elements from items, filtering out the ones that don't match the criteria specified.

Reply:
renamed, thanks!